src/main/java/edu/whimc/portals/listeners/PortalEnterListener.java

```diff
@@ -19,66 +19,105 @@
 import edu.whimc.portals.utils.Messenger;
 import edu.whimc.portals.utils.Messenger.Message;
 
-public class PortalEnterListener implements Listener{
-
+/**
+ * Listens for events where the portal is entered.
+ */
+public class PortalEnterListener implements Listener {
+    /* The list of UUIDs of players with debug mode enabled. */
     private static Set<UUID> debugPlayers = new HashSet<>();
 
+    /**
+     * Adds the UUID of the passed player to hte list of players with debug mode enabled.
```

Comment:
```suggestion
     * Enable debug mode for the given player.
```
You don't want to get too into the specifics of the implementation.